scipy/special/tests/test_basic.py

```diff
@@ -1666,8 +1666,9 @@ def test_euler(self):
         eu0 = special.euler(0)
         eu1 = special.euler(1)
         eu2 = special.euler(2)   # just checking segfaults
-        assert_almost_equal(eu0[0],1,8)
-        assert_almost_equal(eu2[2],-1,8)
+        assert_allclose(eu0, [1], rtol=1e-15)
+        assert_allclose(eu1, [1, 0], rtol=1e-15)
```

Comment:
Note that all of the odd Euler numbers are exactly zero--i.e. as a user I'd be pretty annoyed if instead they were only zero up to (any) small `atol`. (`special.chebyt(10).coeffs` grrr...) I think this is an instance where we want literal floating point equality.